File: TrabajoFinalPrueba1/PegasusV1/PegasusV1/Entities/CuadernoComunicados.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace PegasusV1.Entities
{
    [Table("CUADERNO_COMUNICADOS")]
    public class CuadernoComunicados
    {
        public int Id { get; set; }        

        [ForeignKey("Id_Usuario")]
        public Usuario? Usuario { get; set; }
        public int? Id_Usuario { get; set; }

        [ForeignKey("Id_Curso")]
        public Curso? Curso { get; set; }
        public int? Id_Curso { get; set; }

        public string Descripcion  { get; set; }

        public DateTime? Fecha  { get; set; }
    }
}

File: pegasusapp/src/models/cuadernoComunicados.ts
import { Curso } from './curso';
import { Usuario } from './usuario';

export interface CuadernoComunicados {
  id: number;
  id_Usuario?: number | null;
  id_Curso?: number | null;

  descripcion: string;
  fecha?: string | null;

  usuario?: Usuario | null;
  curso?: Curso | null;
}
